dotnet/content/applicationHost.xdt

```diff
@@ -58,7 +58,8 @@
 
         <add name="DD_DOTNET_TRACER_HOME" value="%XDT_EXTENSIONPATH%\vFOLDERUNKNOWN\Tracer" xdt:Locator="Match(name)" xdt:Transform="Insert"/>
         <add name="DD_INTEGRATIONS" value="%XDT_EXTENSIONPATH%\vFOLDERUNKNOWN\Tracer\integrations.json" xdt:Locator="Match(name)" xdt:Transform="Insert"/>
-        <add name="DD_TRACE_LOG_PATH" value="%HOME%\LogFiles\datadog\vFOLDERUNKNOWN\dotnet-profiler.log" xdt:Locator="Match(name)" xdt:Transform="Insert"/>
+        <add name="DD_TRACE_LOG_PATH" value="%HOME%\LogFiles\datadog\vFOLDERUNKNOWN\dotnet-native-tracer.log" xdt:Locator="Match(name)" xdt:Transform="Insert"/>
```

Comment:
This might be a breaking change. Is it ok or not ?

Reply:
I think it's probably ok in this case, as it's very unlikely there's any automation around this in AAS.

However, I don't think we should set `DD_TRACE_LOG_PATH` at all: that's deprecated. We should set `DD_TRACE_LOG_DIRECTORY` instead, and then you probably don't need to set `DD_PROFILING_LOG_PATH` at all

Reply:
Ok if there is no automation I can replace the `XX_LOG_PATH` by `XX_LOG_DIR`